test/kubemark/common/util.sh

```diff
@@ -18,10 +18,11 @@
 function run-cmd-with-retries {
   RETRIES="${RETRIES:-3}"
   for attempt in $(seq 1 ${RETRIES}); do
+    ret_val=0
```

Comment:
Good catch! Just read about the difference this has wrt variable scope.
Fixed it.